docs/api.md

```
@@ -66,6 +71,13 @@ const updatedBody = validation.ok
   : result.body;
 ```
 
+`extractShepherdJournal(body)` returns the one visible structural journal as
+`{ ok: true, journal: { format, entries } }`, or `{ ok: true, journal: null }` when none exists.
+Each entry retains its leading `- ` marker and continuation indentation, with line endings
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)